frontend/components/notifications/DashboardsTable.vue

```diff
@@ -61,8 +61,8 @@ const { overview } = useNotificationsDashboardOverviewStore()
       <template #table>
         <ClientOnly fallback-tag="span">
           <BcTable
-            :data="notificationsDashboards"
-            data-key="notification_id"
+            :data="wrapWithIdentifier(notificationsDashboards, 'is_account_dashboard', 'dashboard_id', 'group_id', 'epoch')"
+            data-key="wrapped_identifier"
```

Comment:
when using `<BcTable>` and using `wrapedWithIdentifier` function, which has the consequence that `data-key` would be always the `wrapped_identifier` key...

It would be nice if `BcTable` would be refactored that `data-key` is not needed anymore

Reply:
Do we want to refactor `BcTable` to do that though? As far as the prime vue component goes, `data-key` is optional and only optimizes performance, which may be lost if we wrap every time when using `BcTable`. I would suggest investigating whether `BcTable` truly needs a unique ID first.